docker/complete/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM mesosphere/mesos:0.22.0-1.0.ubuntu1404
+MAINTAINER Mesosphere <support@mesosphere.io>
+
+RUN locale-gen en_US.UTF-8
+RUN dpkg-reconfigure locales
+ENV LANG en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+
```

Comment:
Presuming your goal is to stop the stderr red text from showing up when docker is building, I'm adding `DEBIAN_FRONTEND=noninteractive` to JUST the apt-get command.

Based on the below issue, it seems that noninteractive would be wrong if the container is to be used in interactive mode, but is correct for the non-interactive image building.
https://github.com/docker/docker/issues/4032

Unfortunately the UNSET dockerfile command is still in limbo. So we can't just set it at the begining and unset it at the end.
https://github.com/docker/docker/pull/8177